lib/util/cmcd_manager.js

```diff
@@ -55,6 +58,30 @@ shaka.util.CmcdManager = class {
      * @private {boolean}
      */
     this.starved_ = false;
+
+    /**
+     * @private {shaka.extern.RequestFilter}
+     */
+    this.requestFilter_ = (type, request) => {
+      if (!this.config_.enabled || request.method !== 'HEAD') {
+        return;
+      }
+
+      this.apply_(request, {su: false});
```

Comment:
Could you please add a comment about what "su: false" does and why it belongs on all HEAD requests?